src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <div class="logo">
+        <a href="/#home">
+          <img
+            src="./images/logo.png"
+            alt="Company logo"
+        />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list_link">
+            <a
+              href="apple.html"
+              class="nav_link nav_link1 is_active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="list_link">
+            <a
+              href="samsung.html"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="list_link">
+            <a
+              href="smartphones.html"
+              class="nav_link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="list_link">
+            <a
+              href="laptops-computers.html"
+              class="nav_link is_active"
+              data-qa="hover"
+            >
+              laptops&computers
+            </a>
+          </li>
+          <li class="list_link">
+            <a
+              href="gadgets.html"
+              class="nav_link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="list_link">
+            <a
+              href="tablets.html"
+              class="nav_link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="list_link">
+            <a
+              href="photo.html"
+              class="nav_link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="list_link">
+            <a
+              href="video.html"
+              class="nav_link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. For example, use `nav__item` instead of `list_link` and `nav__link` instead of `nav_link`.